src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSFunctionBinding.cs

```diff
@@ -228,7 +228,7 @@ internal static unsafe void InvokeJSFunctionCurrent(JSObject jsFunction, Span<JS
             var functionHandle = (int)jsFunction.JSHandle;
             fixed (JSMarshalerArgument* ptr = arguments)
             {
-                Interop.Runtime.InvokeJSFunction(functionHandle, (nint)ptr);
+                Interop.Runtime.InvokeJSFunction(functionHandle, (void*)ptr);
```

Comment:
Hmm, this is upstream code. Should we rather change the signatures for `InvokeJSFunction`, etc, in `Interop.Runtime.NativeAOT.cs`?

Reply:
Yes, better.  Done.